theano/misc/tests/test_pycuda_theano_simple.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
It should go after the docstring, otherwise the string it is not recognized as a "module-level docstring".